### PR TITLE
Feature/layout styling

### PR DESCRIPTION
**Pull request by mef-bomb**

### CODE REVIEW

File: frontend-new/pages/main/index.tsx [hunk 1]
```diff
@@ -1,31 +1,39 @@
...
   message: string
 }
 
-export default function Home() {
+interface Props {
+  user?: User
+}
+
+export default function Home({ user }: Props) {
   const [notificationState, setNotificationState] = useState<NotificationState>()
```
Comment (maxtechera) [inline frontend-new/pages/main/index.tsx:37]: We should extract this logic into a common place. We can begin by moving this into a custom `useLinks` hook

**Review by maxtechera — APPROVED** [review]
LGTM!
Some comments as we continue building this up. 
Let's combine all the logic on interacting with the go links into one place. 
THis can be a custom hook that we define that returns all the functions and links themselves.

File: frontend-new/pages/_app.tsx [hunk 2]
```diff
@@ -1,34 +1,99 @@
...
+`
 
-  .main {
-    height: calc(100% - 64px - 64px);
-  }
+const Main = styled.div`
+  height: calc(100% - 64px - 64px);
 `
 
 const theme = createTheme({
```
[the remark this answers is not in the document]
Comment (mef-bomb) [reply frontend-new/pages/_app.tsx:19]: sure, it is already big enough :)

File: frontend-new/pages/main/index.tsx (hunk 1 above)
Comment (mef-bomb) [reply frontend-new/pages/main/index.tsx:37]: just for this component? Or are you meaning to have a hook that can be reused in other components and incapsulates `useContext` inside?

**Pull request merged**